website/src/docs/hotchocolate/server/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Overview
+---
+
+In this section we will learn how we can configure our GraphQL server and look at some transport protocol related details.
+
+# Middlewares
+
+Hot Chocolate comes with ASP.NET Core middlewares for accepting HTTP / WebSocket GraphQL requests, downloading the GraphQL schema or serving the [Banana Cake Pop](/docs/bananacakepop) GraphQL IDE.
```

Comment:
```suggestion
Hot Chocolate comes with ASP.NET Core middlewares for accepting HTTP / WebSocket GraphQL requests, downloading the GraphQL schema, or serving the [Banana Cake Pop](/docs/bananacakepop) GraphQL IDE.
```

Reply:
I believe the plural of middleware is just middleware. Just like software --> software. 

Worth double checking but it feels weird to say middlewares. It's in the title and this sentence. Maybe other places.